### PR TITLE
fix(ui): Missing french translations dt details panel

### PR DESCRIPTION
**Pull request by Jordan91**
## Description

When UI is in french, some sentences or words are not translated in popins and timeline.

## Type of change

- [x] Patch fixing an issue (non-breaking change)

## Target serie

- [x] 20.10.x
- [x] 21.04.x (master)

<h2> How this pull request can be tested ? </h2>

Log on centreon , set language in french and set an Acknowledge and set a Downtime.
After you can check in popins , timeline and notify option all translations are OK.

### CODE REVIEW

File: www/front_src/src/Resources/translatedLabels.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const labelCancel = 'Cancel';
 export const labelClear = 'Clear';
 export const labelCriterias = 'Criterias';
 export const labelCritical = 'Critical';
-export const labelCheckDuration = 'Check duration';
+export const labelCheckDuration = 'Check Duration';
```
Comment (Thebarda) [inline www/front_src/src/Resources/translatedLabels.ts:15]:
```suggestion
export const labelCheckDuration = 'Check duration';
```

Comment (Jordan91) [reply www/front_src/src/Resources/translatedLabels.ts:15]: Ok , i'll do modification into four files messages.po

File: lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -14645,4 +14645,7 @@ msgstr "Permitir certificado autofirmado"
 #  msgstr ""
 
 #  msgid "Monitoring server"
-#  msgstr ""
\ No newline at end of file
+#  msgstr ""
+
+msgid "Check duration"
+msgstr "Temps d'exécution"
```
Comment (Thebarda) [inline lang/es_ES.UTF-8/LC_MESSAGES/messages.po:14651]:
```suggestion
msgstr ""
```

File: lang/pt_BR.UTF-8/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -15088,4 +15088,7 @@ msgstr "Erro ao remover tokens de um contato"
 #  msgstr ""
 
 #  msgid "Monitoring server"
-#  msgstr ""
\ No newline at end of file
+#  msgstr ""
+
+msgid "Check duration"
+msgstr "Temps d'exécution"
```
Comment (Thebarda) [inline lang/pt_BR.UTF-8/LC_MESSAGES/messages.po:15094]:
```suggestion
msgstr ""
```

File: lang/pt_PT.UTF-8/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -15089,4 +15089,7 @@ msgstr "Erro ao remover tokens de um contato"
 #  msgstr ""
 
 #  msgid "Monitoring server"
-#  msgstr ""
\ No newline at end of file
+#  msgstr ""
+
+msgid "Check duration"
+msgstr "Temps d'exécution"
```
Comment (Thebarda) [inline lang/pt_PT.UTF-8/LC_MESSAGES/messages.po:15095]:
```suggestion
msgstr ""
```

File: www/front_src/src/Resources/Listing/columns/State/DetailsTable/Downtime.tsx [hunk 5]
```diff
@@ -55,7 +55,7 @@ const DowntimeDetailsTable = ({ endpoint }: Props): JSX.Element => {
       id: 'is_fixed',
       label: t(labelFixed),
       type: ColumnType.string,
-      getContent: ({ is_fixed }): string => (is_fixed ? labelYes : labelNo),
+      getContent: ({ is_fixed }): string => t(is_fixed ? labelYes : labelNo),
```
Comment (kduret) [inline www/front_src/src/Resources/Listing/columns/State/DetailsTable/Downtime.tsx:58]: you could use `getYesNoLabel` to be consistent with acknowledgement

Comment (Jordan91) [reply www/front_src/src/Resources/Listing/columns/State/DetailsTable/Downtime.tsx]: Well done ! I did it !    :)

**Comment by Jordan91** [comment]
Acknowledgement
Downtime

Il reste la documentation à traduire

**Review by kduret — APPROVED** [review]

**Review by jdelpierre — APPROVED** [review]

**Review by bdauria — APPROVED** [review]

**Pull request merged**